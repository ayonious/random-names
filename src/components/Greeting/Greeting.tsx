import React from 'react';

import { CenteredFlexDevWrapper } from './styles';

const Greeting = () => {
  return (
    <CenteredFlexDevWrapper>
      <h1> Start Shuffling! </h1>
    </CenteredFlexDevWrapper>
  );
};

export default Greeting;
