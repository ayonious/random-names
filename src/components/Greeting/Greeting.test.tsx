/**
 * @jest-environment jsdom
 */
import { expect } from '@jest/globals';
import { render } from '@testing-library/react';
import React from 'react';

import Greeting from '.';

it('Greeting renders', () => {
  const { container } = render(<Greeting />);
  expect(container).toMatchSnapshot();
});
