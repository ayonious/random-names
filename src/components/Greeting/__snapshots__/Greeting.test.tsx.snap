// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting renders 1`] = `
<div>
  <div
    class="sc-bXdtCk lfUsMd"
  >
    <h1>
       Start Shuffling! 
    </h1>
  </div>
</div>
`;
